indico/modules/events/abstracts/util.py

```diff
@@ -124,7 +124,8 @@ def _build_query(self):
                          subqueryload('submitted_for_tracks'),
                          subqueryload('reviewed_for_tracks'),
                          subqueryload('person_links'),
-                         subqueryload('reviews').joinedload('ratings'))
+                         subqueryload('reviews').joinedload('ratings'),
+                         subqueryload('contribution'))
```

Comment:
I would use `joinedload('contribution').load_only('id', 'event_id')` - you don't need anything but the IDs from the contribution since all you do is building a URL, and that way 1) don't query everything and 2) don't use a seperate query (which is the case with `subqueryload`) to get two extra columns (the reason why some of the others are `subqueryload` is that they are relationships containing multiple rows - the ones where you get zero or one extra row are all `joinedload`)